akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSupervisionSpec.scala

```diff
@@ -387,70 +387,6 @@ class InterpreterSupervisionSpec extends AkkaSpec with GraphInterpreterSpecKit {
       lastEvents() should be(Set(OnNext(25))) // 1 + 4 + 20
     }
 
-    "restart when Conflate `seed` throws" in new OneBoundedSetup[Int](Seq(Conflate(
```

Comment:
How about I remove those tests but add coverage for supervision in the `FlowBatchSpec` instead?